ledger-api/grpc-definitions/BUILD.bazel

```diff
@@ -121,15 +127,15 @@ genrule(
         here=$$(dirname $(locations BUILD.bazel))
         well_known=$$(dirname $$(dirname $$(dirname $$(echo $(locations @com_google_protobuf//:well_known_protos) | cut -d ' ' -f1))))
         rpc_status=$$(dirname $$(dirname $$(dirname $$(echo $(location @com_github_googleapis_googleapis//google/rpc:status.proto)))))
-        for src in $$here/com/digitalasset/ledger/api/v1/*.proto; do 
+        for src in $(locations :ledger-api-protos-fg); do
```

Comment:
@nickchapman-da I modified the genrule you created recently as it was not taking into account `srcs` you were setting, but rather compiling all protos. Not sure why linux/mac builds were green taking into account your `TODO` comments, but it was failing on Windows for these commented out protos, so I modifed the rule to compile only these explicitly mentioned in `srcs`.